docs/services/aws-services-guide.md

```diff
@@ -506,212 +239,101 @@ that we configured earlier.
+including referencing the class or managed resource directly, providing labels
+that are used to match to a class, or by defaulting to a class that is annotated
+with `resourceclass.crossplane.io/is-default-class: "true"`. In the
+`RDSInstanceClass` above, we added the default annotation, so our claim will
+bind to it automatically if no other classes exist with annotation. If there are
+multiple classes annotated as default, one will be chosen at random.
 
-You may specify *one* instance of a portable class kind as *default* in each
-namespace. This means that the portable resource class instance will be applied
-to claims that do not directly reference a portable class. If we wanted to make
-our `mysql-standard` instance the default `MySQLInstanceClass` for namespace
-`app-project1-dev`, we could do so by adding a label:
+* Define a `MySQLInstance` claim in `mysql-claim.yaml` and create it:
 
-```yaml
----
+```bash
+cat > mysql-claim.yaml <<EOF
 apiVersion: database.crossplane.io/v1alpha1
-kind: MySQLInstanceClass
+kind: MySQLInstance
 metadata:
-  name: mysql-standard
-  namespace: app-project1-dev
-  labels:
-    default: "true"
-classRef:
-  kind: RDSInstanceClass
-  apiVersion: database.aws.crossplane.io/v1alpha2
-  name: aws-mysql-standard
-  namespace: aws-infra-dev
-```
-
-#### Resource Claims
-
-Resource claims are used to create external resources by referencing a class of
-service in the claim's namespace. When a claim is created, Crossplane uses the
-referenced portable class to find a cloud-specific resource class to use as the
-configuration for the external resource. We need a to create a claim to
-provision the RDS database we will use with AWS.
-
-* Define a `MySQLInstance` claim in `mysql-claim.yaml` and create it:
+  name: mysql-claim
+spec:
+  writeConnectionSecretToRef:
+    name: wordpressmysql
+  engineVersion: "5.6"
+EOF
 
-  ```yaml
-  cat > mysql-claim.yaml <<EOF
-  apiVersion: database.crossplane.io/v1alpha1
-  kind: MySQLInstance
-  metadata:
-    name: mysql-claim
-    namespace: app-project1-dev
-  spec:
-    classRef:
-      name: mysql-standard
-    writeConnectionSecretToRef:
-      name: wordpressmysql
-    engineVersion: "5.6"
-  EOF
-
-  kubectl apply -f mysql-claim.yaml
-  ```
+kubectl apply -f mysql-claim.yaml
+```
 
 What we are looking for is for the `STATUS` value to become `Bound` which
 indicates the managed resource was successfully provisioned and is ready for
 consumption. You can see when claim is bound using the following:
 
 ```bash
-$ kubectl get mysqlinstances -n app-project1-dev
-NAME          STATUS   CLASS            VERSION   AGE
-mysql-claim   Bound    mysql-standard   5.6       11m
+$ kubectl get mysqlinstances
+NAME          STATUS   CLASS-KIND         CLASS-NAME       RESOURCE-KIND   RESOURCE-NAME               AGE
+mysql-claim            RDSInstanceClass   standard-mysql   RDSInstance     default-mysql-claim-5p66w   9s
```

Comment:
The paragraph leading into this example implies we probably want it to say `Bound` under the `STATUS` column.